mmv1/templates/terraform/examples/gkehub_membership_rbac_role_binding_basic.tf.erb

```diff
@@ -0,0 +1,27 @@
+resource "google_container_cluster" "primary" {
+  name               = "<%= ctx[:vars]['cluster_name'] %>"
+  location           = "us-central1-a"
+  initial_node_count = 1
+}
+
+resource "google_gke_hub_membership" "<%= ctx[:primary_resource_id] %>" {
+  membership_id = "tf-test-membership%{random_suffix}"
+  endpoint {
+    gke_cluster {
+      resource_link = "//container.googleapis.com/${google_container_cluster.primary.id}"
+    }
+  }
+  
+  depends_on = [google_container_cluster.primary]
+}
+
+resource "google_gke_hub_membership_rbac_role_binding" "<%= ctx[:primary_resource_id] %>" {
+  membership_rbac_role_binding_id = "tf-test-membership-rbac-role-binding%{random_suffix}"
+  membership_id = "tf-test-membership%{random_suffix}"
+  user = "service-155926358489@gcp-sa-anthossupport.iam.gserviceaccount.com"
```

Comment:
Is this value stable? I worry about an external dependency here.
```
  user = "service-155926358489@gcp-sa-anthossupport.iam.gserviceaccount.com"
  ```

Reply:
That's a great catch, thank you.

Reply:
I have asked a YAQS question for this, and still figuring out how to fix this. Thank you

Reply:
https://yaqs.corp.google.com/eng/q/7914452922010697728

Reply:
Made a change. Please review it. Thank you